package.json

```diff
@@ -78,7 +83,8 @@
     "restoreMocks": true,
     "collectCoverageFrom": [
       "src/**",
-      "!src/components/Address/i18n.ts"
+      "!src/components/Address/i18n.ts",
+      "!src/components/Markdown.tsx"
```

Comment:
Should we be omitting this component from coverage, even if we're planning to add tests later? There's some mental baggage with this since the person who adds the tests has to remember to add this file back to coverage.

Reply:
Because React Markdown is ESM-only, Jest can't collect coverage from it without a transform. Getting the transform right/working has proved difficult. I'll be making another item to add Jest coverage and undoing this line will be called out in the item description 👍

Reply:
Oh ok gotcha